src/components/Button/Button.module.css

```diff
@@ -121,6 +121,106 @@
   }
 }
 
+/**
+ * Secondary button
+ * 1) Medium prominence button used for less important actions
+ */
+.button--secondary {
+  border-color: var(--eds-theme-color-primary-border);
+  background-color: var(--eds-theme-color-primary-background);
+  color: var(--eds-theme-color-body-foreground-inverted);
+
+  &:hover {
+    background-color: var(--eds-theme-color-primary-background-hover);
+    color: var(--eds-theme-color-body-foreground-inverted);
+  }
+
+  &:focus {
+    background-color: var(--eds-theme-color-primary-background-hover);
+    color: var(--eds-theme-color-body-foreground-inverted);
+  }
+
+  &:focus-visible {
+    @mixin focus;
+  }
+
+  &:disabled {
+    background-color: var(--eds-theme-color-disabled-background);
+    border-color: var(--eds-theme-color-disabled-border);
+    color: var(--eds-theme-color-disabled-foreground);
+    /* stylelint-disable-next-line max-nesting-depth */
+    &:hover {
```

Comment:
to DRY some (since these are the exact same styles), could this be
```
&:disabled,
&:disabled:hover {
```

Reply:
Done, thanks!